src/helpers/httpRequest.ts

```diff
@@ -0,0 +1,18 @@
+export function httpRequest(method: httpMethod, url: string, payload?: {}) {
+  const headers = {
+    Accept: 'application/json',
+  };
+  let body;
+  if (method !== httpMethod.get && method !== httpMethod.delete) {
+    headers['Content-Type'] = 'application/json';
+    body = JSON.stringify(payload);
+  }
+  return fetch(`http://localhost:1111/${url}`, { headers, method, body });
+}
+
+export enum httpMethod {
+  get = 'GET',
+  post = 'POST',
+  delete = 'DELETE',
+  patch = 'PATCH',
+}
```

Comment:
Kudos for creating the wrapper around fetch by hand.
You could alternatively use `axios` instead which helps eliminate most of the fetch-related boilerplate.
The article I once told you about:
https://medium.com/@shahata/why-i-wont-be-using-fetch-api-in-my-apps-6900e6c6fe78

Reply:
This is not something you need to fix, just an alternative to read about.
The MR itself is fine to be merged.